newIDE/app/src/UI/SearchBar.js

```diff
@@ -96,141 +109,266 @@ const getStyles = (props: Props, state: State) => {
-                hintText={this.props.placeholder || t`Search`}
-                onBlur={this.handleBlur}
-                value={value}
-                onChange={this.handleInput}
-                onKeyUp={this.handleKeyPressed}
-                onFocus={this.handleFocus}
-                fullWidth
-                style={styles.input}
-                underlineShow={false}
-                disabled={disabled}
-                ref={this._textField}
-              />
-            </div>
-            {buildTagsMenuTemplate && (
-              <ElementWithMenu
-                element={
-                  <IconButton
-                    style={styles.iconButtonFilter.style}
-                    disabled={disabled}
-                    size="small"
+      <I18n>
+        {({ i18n }) => (
+          <ThemeConsumer>
+            {muiTheme => (
+              <Column>
+                <Line>
+                  <Paper
+                    style={{
+                      backgroundColor: muiTheme.searchBar.backgroundColor,
+                      ...styles.root,
+                      ...style,
+                    }}
+                    square
+                    elevation={1}
                   >
-                    <FilterList />
-                  </IconButton>
-                }
-                buildMenuTemplate={buildTagsMenuTemplate}
-              />
-            )}
-            {helpPagePath && (
-              <HelpIcon
-                disabled={disabled}
-                helpPagePath={helpPagePath}
-                style={styles.iconButtonHelp.style}
-                size="small"
-              />
+                    <div style={styles.searchContainer}>
+                      {tags ? (
+                        <Autocomplete
+                          options={tags}
+                          groupBy={options => i18n._(t`Apply a filter`)}
+                          classes={autocompleteStyles}
+                          freeSolo
+                          fullWidth
+                          defaultValue=""
+                          inputValue={value}
+                          value={autocompleteValue}
+                          onChange={handleAutocompleteInput}
+                          onInputChange={handleAutocompleteInputChange}
+                          onKeyPress={handleKeyPressed}
+                          onBlur={handleBlur}
+                          renderOption={option => (
+                            <Typography>{option}</Typography>
+                          )}
+                          renderInput={params => (
+                            <MuiTextField
+                              margin="none"
+                              {...params}
+                              inputRef={textField}
+                              InputProps={{
+                                ...params.InputProps,
+                                disableUnderline: true,
+                                endAdornment: null,
```

Comment:
I guess you can remove this line?

Reply:
I need it to make sure TextField doesn't inject its own adornments